centreon/packages/ui/src/Form/FormButtons.tsx

```diff
@@ -58,13 +58,16 @@ const FormButtons = (): JSX.Element => {
       <div className={classes.buttons}>
         <Button
           aria-label={t(labelReset)}
+          data-testid={labelReset}
           disabled={not(canReset)}
+          id={labelReset?.replace(/[^A-Z0-9]+/gi, '')}
```

Comment:
Why is that needed? can we create a util function in order to reuse the id normalization?

Reply:
@SoldierBoy31

Reply:
Agree, I will create it in utils folder but what type of function should i use "getNormalizedId" or useNormalizedId ?

Reply:
If you want a function, pick up the "getNormalizedId". If you want a custom hook, pick the other one